### ISSUE TITLE
await tag and webpack server rendering

### ISSUE THREAD

**Issue by arthurchenn** [marko-js/marko#728]
When using <await> tag in server rendering with webpack, I got this error:
```
node_modules/raptor-logging/lib/raptor-logging-impl.js:128
            if (loggerName.startsWith(curLoggerName)) {
                           ^

TypeError: loggerName.startsWith is not a function
```
Using <await> in browser have no problem.

**Comment by arthurchenn** [comment]
This is a project on github to reproduce the problem:
https://github.com/arthurchenn/markoissue728
@austinkelleher could you please take a look at this.
btw, thanks for the help on issue702.

**Comment by austinkelleher** [comment]
@arthurchenn Thanks for the project that reproduces the problem. I'll take a look.

**Comment by austinkelleher** [comment]
Fixed and published `marko@4.4.10`. Thanks for opening up this issue.

**Comment by arthurchenn** [comment]
Thanks for the quick response.
